### PR TITLE
Add alt text to images

### LINKED ISSUE #334: No alt text on main banner logo

### LINKED ISSUE #335: No alt text on placecal footer svg

### LINKED ISSUE #336: map is missing alt text

### LINKED ISSUE #337: Missing alt text on map

### PR DESCRIPTION
**Pull request by erbridge**
Fixes #337 
Fixes #336 
Fixes #335 
Fixes #334

### CODE REVIEW

File: src/Page/Events/Event_.elm [hunk 1]
```diff
@@ -113,7 +113,11 @@ viewEventInfo event =
         , case event.maybeGeo of
             Just geo ->
                 div [ css [ mapContainerStyle ] ]
-                    [ p [] [ Theme.Global.mapImage { latitude = geo.latitude, longitude = geo.longitude } ]
+                    [ p []
+                        [ Theme.Global.mapImage
+                            ("A map showing the location of " ++ event.name)
```
Comment (katjam) [inline src/Page/Events/Event_.elm:118]: I'd like all strings to go through the `Copy.Text.t` function. 

This includes alt text. I won't flag all of those up in this PR individually.

This 1) makes it easier to localise & 2) makes it easy to edit UI copy in one place (good overview / visibility on public facing content)

**Review by katjam — CHANGES_REQUESTED** [review]
Unless there is a compelling argument otherwise, we should stick to piping all copy through translate function.

File: src/Page/Events/Event_.elm (hunk 1 above)
Comment (erbridge) [reply src/Page/Events/Event_.elm:118]: I did wonder about this. Happy to do this, sure.

Comment (erbridge) [reply src/Page/Events/Event_.elm]: This should be done now.

**Review by katjam — APPROVED** [review]
This is looking good now.

File: src/Theme/Global.elm [hunk 2]
```diff
@@ -830,14 +830,24 @@ generateId input =
...
+        []
 
 
-mapImage : { latitude : String, longitude : String } -> Html msg
-mapImage geo =
-    img [ src ("https://api.mapbox.com/styles/v1/studiosquid/cl082tq5a001o14mgaatx9fze/static/pin-l+ffffff(" ++ geo.longitude ++ "," ++ geo.latitude ++ ")/" ++ geo.longitude ++ "," ++ geo.latitude ++ ",15,0/1140x400@2x?access_token=pk.eyJ1Ijoic3R1ZGlvc3F1aWQiLCJhIjoiY2o5bzZmNzhvMWI2dTJ3bnQ1aHFnd3loYSJ9.NC3T07dEr_Aw7wo1O8aF-g"), css [ mapStyle ] ] []
+mapImage : String -> { latitude : String, longitude : String } -> Html msg
+mapImage altText geo =
+    img
+        [ src ("https://api.mapbox.com/styles/v1/studiosquid/cl082tq5a001o14mgaatx9fze/static/pin-l+ffffff(" ++ geo.longitude ++ "," ++ geo.latitude ++ ")/" ++ geo.longitude ++ "," ++ geo.latitude ++ ",15,0/1140x400@2x?access_token=pk.eyJ1Ijoic3R1ZGlvc3F1aWQiLCJhIjoiY2o5bzZmNzhvMWI2dTJ3bnQ1aHFnd3loYSJ9.NC3T07dEr_Aw7wo1O8aF-g")
```
Comment (ivan-kocienski-gfsc) [inline src/Theme/Global.elm:846]: Could this URL be moved into a constant? Or could this string concatenation be moved into a helper method?

**Pull request merged**